admin/workspace-providers/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Workspace Providers
+description: Learn how workspace providers can minimize latency for users.
```

Comment:
Thoughts on something along the lines of:

```
Learn how workspace providers empower distributed organizations
```

Reply:
> Learn how workspace providers can improve developer experience and security

?

Is security too bold a claim for isolating environments from one another?

Reply:
I like yours!